api/v1beta1/azuremanagedcontrolplane_webhook.go

```diff
@@ -709,23 +709,23 @@ func (m *AzureManagedControlPlane) validateVirtualNetworkUpdate(old *AzureManage
 	if old.Spec.VirtualNetwork.Name != m.Spec.VirtualNetwork.Name {
 		allErrs = append(allErrs,
 			field.Invalid(
-				field.NewPath("Spec", "VirtualNetwork.Name"),
+				field.NewPath("spec", "virtualNetwork.name"),
```

Comment:
```suggestion
				field.NewPath("spec", "virtualNetwork", "name"),
```
The same applies for the rest of the virtualNetwork fields.